RyanNoelk/OpenEats
# Fix/filters

Implements a new search function that mirrors the Django one so the left rail filters match the results returned. 

To Test:
- Play around with search and filter and make sure everything is returned correctly.
